psi4/src/psi4/libfock/v.cc

```diff
@@ -2886,9 +2907,9 @@ void UV::compute_Vx(std::vector<SharedMatrix> Dx, std::vector<SharedMatrix> ret)
             rho_bk_y = R_rho_bk_y[rank]->pointer();
             rho_bk_z = R_rho_bk_z[rank]->pointer();
             gamma_bbk = R_gamma_bk[rank]->pointer();
-            rho_bx = pworker->point_value("RHO_AX")->pointer();
-            rho_by = pworker->point_value("RHO_AY")->pointer();
-            rho_bz = pworker->point_value("RHO_AZ")->pointer();
+            rho_bx = pworker->point_value("RHO_BX")->pointer();
+            rho_by = pworker->point_value("RHO_BY")->pointer();
+            rho_bz = pworker->point_value("RHO_BZ")->pointer();
```

Comment:
uhm this looks like a pretty important bug fix...?

Reply:
Thanks, I forgot about this one and will edit the main post accordingly.

`UV::compute_Vx` wasn't correct for GGAs. The two parts of Psi that might use this function, TDDFT and stability analysis, both refuse to run for UKS GGAs, which is why Psi hasn't been producing wrong results. The lines in question fix an obvious bug in that function, but the function _still_ isn't correct. When I enable UTDDFT with GGAs, the old behavior was positive semidefinite violations. The new behavior is convergence failure. This suggests another bug.

Once this PR is in, I'll do a similar commenting of `compute_Vx` up to GGAs. The process of doing that will presumably uncover the bug, and I can re-enable TDDFT and stability analysis for unrestricted GGAs.

There are no other lines in this PR that will change numerical values.

Reply:
oh wow!